trymito.io/pages/mito_pro_roadmap.tsx

```diff
@@ -0,0 +1,158 @@
+import Header from '../components/Header/Header';
+import securityStyles from '../styles/Security.module.css'
+import pageStyles from '../styles/Page.module.css'
+import titleStyles from '../styles/Title.module.css'
+import GithubButton from '../components/GithubButton/GithubButton';
+import DownloadCTACard from '../components/CTACards/DownloadCTACard';
+
+import FlagIcon from '../public/icon-squares/FlagIcon.svg'
+
+
+const Mito_Pro_Roadmap: NextPage = () => {
+
+  return (
+    <>
+      <Head>
+        <title>Mito | Roadmap </title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+      </Head>
+      
+      <Header/>
+    
+      <div className={pageStyles.container}>
+
+        <main className={pageStyles.main}>
+
+            <section className={titleStyles.title_card + ' ' + titleStyles.grid_card}>
+                <h1 className={titleStyles.title}>
+                    Mito Pro Roadmap 
+                </h1>
+                <p className={titleStyles.description}>
+                    We&apos;re constantly releasing new features to speed up your data analysis
+                </p>
+            </section>
+            <section>
+                <div className={pageStyles.subsection}>
+                    <div className={securityStyles.security_bullet_container}>
+                        <div className={securityStyles.icon}>
+                            <Image className={securityStyles.icon} src={FlagIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Shareable <br/> notebooks
+                        </h1>
+                        <p>
+                            Share notebooks with Mito embedded in them and let colleagues use the Mito Spreadsheet too.  
+                        </p>
+                    </div>
+                    <div className={securityStyles.security_bullet_container + ' ' + pageStyles.subsection_second_element_mobile_spacing}>
+                        <div className={securityStyles.icon}>
+                            <Image className={securityStyles.icon} src={FlagIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Advanced <br/> analysis
+                        </h1>
+                        <p>
+                            Go beyond basic data cleaning and analysis features with support for regressions, fuzzy matching and clustering. 
+                        </p>
+                    </div>
+                </div>
+                <div className={pageStyles.subsection}>
+                    <div className={securityStyles.security_bullet_container}>
+                        <div className={securityStyles.icon}>
+                            <Image className={securityStyles.icon} src={FlagIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Custom steps and functions
```

Comment:
I don't think people know what steps are. Let's just call this "Custom transformations" or "Custom data transformations" or something...